NGS2_WITNESS_Cycle1_MASTER.R

```diff
@@ -0,0 +1,59 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. on August 2017
+
+### System information
+
+a=Sys.info()
+print(a)
+b=sessionInfo(package = NULL)
+print(a)
+
+### Set base directory
+
+setwd(X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1)

```

Comment:
Thanks Anu! I would like to make sure that we are working off of the X drive though.